starknet_py/tests/e2e/block_test.py

```diff
@@ -30,19 +30,18 @@ async def test_latest_block(account, map_compiled_contract):
     assert blk.block_hash
 
 
+@pytest.mark.parametrize("block_number", ("pending", "latest"))
+@pytest.mark.usefixtures("map_contract_declare")
```

Comment:
Why this line instead of test argument?

Reply:
As an argument it isn't used and pylint throws an error, however it's fine if we want to stay consistent with usage as an argument

Reply:
Let's maybe stay consistent. It's harder to see what's going on once there are 3 decorators 🙈